brainiak/utils/fmrisim.py

```diff
@@ -443,11 +447,9 @@ def generate_stimfunction(onsets,
     if len(weights) == 1:
         weights = weights * len(onsets)
 
-    # How many elements per second are you modeling
-    temporal_resolution = 1000
-
-    # Generate the time course as empty, each element is a millisecond
-    stimfunction = [0] * round(total_time * temporal_resolution)
+    # Generate the time course as empty, each element is a millisecond by
+    # default
+    stimfunction = [0] * round(int(total_time * temporal_resolution))
```

Comment:
I implemented this change because I was getting the type error in Jenkins (which happened again this time). Do you know why it is saying that this is a float? I know that total_time is an, int and temporal_resolution is a float but they seem to play nicely with each other.